lib/r10k/forge/module_release.rb

```diff
@@ -41,9 +46,14 @@ def initialize(full_name, version)
         # objects are created in the class instances and thus are not shared with
         # subclasses.
         PuppetForge::V3::Release.conn = PuppetForge::V3::Base.conn
+
         @forge_release = PuppetForge::V3::Release.new({ :name => @full_name, :version => @version, :slug => "#{@full_name}-#{@version}" })
 
-        @download_path = Pathname.new(Dir.mktmpdir) + (@forge_release.slug + '.tar.gz')
+        tarball_name = @forge_release.slug + '.tar.gz'
+        @download_path = Pathname.new(Dir.mktmpdir) + (tarball_name)
+        @tarball_cache_root = Pathname.new(settings[:cache_root]) + (@forge_release.slug + "/tarball/")
```

Comment:
Does this variable need an `attr_accessor` above too? (as with https://github.com/puppetlabs/r10k/pull/729/files#diff-41ce558d2e9e83cf146b0fad6c9811d6R33)